service/src/test/java/tds/session/repositories/impl/SessionRepositoryImplIntegrationTests.java

```diff
@@ -1,51 +1,129 @@
 package tds.session.repositories.impl;
 
+import org.joda.time.Instant;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.transaction.annotation.Transactional;
 
+import javax.sql.DataSource;
 import java.text.ParseException;
-import java.time.Instant;
 import java.util.Optional;
 import java.util.UUID;
 
 import tds.session.Session;
 import tds.session.repositories.SessionRepository;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static tds.common.data.mapping.ResultSetMapperUtility.mapInstantToTimestamp;
+import static tds.common.data.mysql.UuidAdapter.getBytesFromUUID;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest
 @Transactional
 public class SessionRepositoryImplIntegrationTests {
+    private final static String sessionInsertSQL = "insert into session (\n" +
```

Comment:
This tests was written a while back and still relied on the migration scripts inserting data.  It now inserts and queries in the test itself.